src/main/java/com/microsoft/azure/vmagent/AzureVMAgent.java

```diff
@@ -205,6 +213,8 @@ public AzureVMAgent(
             boolean executeInitScriptAsRoot,
             boolean doNotUseMachineIfInitFails,
             boolean enableMSI,
+            boolean enableUAMI,
```

Comment:
@gavinfish What is this constructor used for? Was it to maintain compatibility or some other reason? (there's no javadoc on it)

not sure if this change is needed here...